debian/clearwater-etcd.prerm

```diff
@@ -64,6 +64,9 @@ case "$1" in
         # Make sure the process is stopped
         service clearwater-etcd stop
 
+        # Clean up old pidfile
+        rm -f /var/run/$NAME.pid
```

Comment:
Surely it would make more sense to remove this on install then (as that's the earliest you can do it)?